components/cart/cart.css

```diff
@@ -0,0 +1,130 @@
+@import url('../../style.css');
+
+.main-container{
+    margin-top: 1.5rem;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-color: var(--gray-bg-color);
+}
+
+.cart-container{
+    display: flex;
+    padding: 2rem 2rem;
+}
+
+.selected-items-container{
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    background-color: var(--white-color);
+    padding: 2rem 3rem;
+    margin: 0 2rem 2rem 2rem;
+}
+
+.product{
+    display: flex;
+    line-height: 1.7;
+    padding: 1rem;
+    margin: 1rem 1rem;
+    width: auto;
+    height: auto;
+    border-top: 1px solid #DDD;
+}
+
+.item-quantity{
+    display: flex;
+    gap: 10px;
+    margin-top: 5px;
+}
+
+.item-quantity button{
+    padding: 0 6px;
+}
+
+.wishlist-icon{
+    font-size: 1.2rem;
+}
+
+.product-price:hover{
+    color: var(--primary-color);
+}
+
+.product-img{
+    width: 25rem;
+    transition: .4s all ease-in-out;
+}
+
+.product-img:hover{
+    transform: scale(1.05);
+}
+
+.product-details{
+    display: flex;
+    flex-direction: column;
+}
+
+.product-desc{
+    font-size: 1.4rem;
+    font-weight: 600;
+}
+
+.product-links{
+    margin-top: 12px;
+}
+
+.product-btn{
+    background-color: var(--primary-color);
+    color: var(--white-color);
+    text-decoration: none;
+    padding: 5px 10px;
+    border-radius: 10px;
+    transition: .5s ease-in-out;
+}
```

Comment:
We have create button in component library, we can you that directly here, and then modify only required properties like color.